running-coreos/index.md

```diff
@@ -0,0 +1,35 @@
+Please follow the instruction below to know how to verify CoreOS images.
+
```

Comment:
First stab at an overview section:

CoreOS publishes new images for each release across a variety of platforms and hosting providers. Each channel has it's own set of images ([stable](http://stable.release.core-os.net/amd64-usr/current/), [beta](http://beta.release.core-os.net/amd64-usr/current/), [alpha](http://alpha.release.core-os.net/amd64-usr/current/)) that are posted to our storage site. Along with each image, a signature is generated from the [CoreOS Image Signing Key](https://coreos.com/security/image-signing-key/) and posted.

After downloading your image, you should verify it with `gpg`. First, download the image signing key: